lib/hawkular/alerts/alerts_api.rb

```diff
@@ -210,6 +210,22 @@ def list_events(*criteria)
       query = generate_query_params(*criteria)
       http_get('/events' + query).map { |e| Event.new(e) }
     end
+
+    # Inject an event into Hawkular-alerts
+    # @param [String] id Id of the event must be unique
+    # @param [String] category Event category for further distinction
+    # @param [String] text Some text to the user
+    # @param [Hash<String,Object>] context Additional context information.
+    def create_event(id, category, text, context = {})
+      event = {}
+      event['id'] = id
+      event['ctime'] = Time.now.to_i * 1000
+      event['category'] = category
+      event['text'] = text
+      event['context'] = context unless context.nil?
```

Comment:
@pilhuhn I'm OK with any impl you choose as long as you extend the inject service to allow initial tags on the new event.  Without tags it is unlikely the event will be picked up by the event catcher given changes coming in the near future.